apps/epic-react/src/pages/learn.tsx

```diff
@@ -31,14 +31,18 @@ const ResourceLink: React.FC<{
   workshopSlug: string
   resourceSlug: string
   isCompleted: boolean
-}> = ({title, workshopSlug, resourceSlug, isCompleted}) => {
+  isBonusModule?: boolean
+}> = ({title, workshopSlug, resourceSlug, isCompleted, isBonusModule}) => {
   const [isHovered, setHovered] = React.useState<Boolean>(false)
+  const resourceLink = isBonusModule
+    ? `/bonuses/${workshopSlug}/${resourceSlug}`
+    : `/workshops/${workshopSlug}/${resourceSlug}`
```

Comment:
Both bonuses and workshops should be "/modules/" eventually to be consistent with original ER:
![CleanShot 2024-04-10 at 23 58 01@2x](https://github.com/skillrecordings/products/assets/1519448/77114541-5ce5-49cd-8538-c8e8a69d8833)